### ISSUE TITLE
Hemostat does not work during organ removal surgery

### ISSUE THREAD

**Issue by PoopyNooges** [Foundation-19/Foundation-19#610]
#### Description of issue
When using an hemostat to remove an organ after separating them, it does not happen, nothing happens or show up, not even a message stating the procedure failed or a small UI which indicates which organ is to be removed. It results in round removal if not intervened by moderators or admins, and can only be fixed by admin console via rejuvenation command.

#### Difference between expected and actual behavior
Expected behavior: Organs will be successfully removed, and the message "organ.name can be seen sticking out" will dissappear when examining characters.
Actual behavior: When using hemostat, nothing happens, no UIs which appears to show which organ to remove first, nor is there any message

#### Steps to reproduce
1. Receive and replicate an organ failure condition which requires replacement
2. Follow the usual steps of organ removal surgery (Scalpel, retractor, hemostat, bonesaw for chest or head, scalpel to separate organ)
3. The final step of the procedure, which uses hemostat to remove the organ will not display any UI or messages, nothing will happen, which results in failure to remove organ.

#### Length of time in which bug has been known to occur
 
It has been about a few weeks this issue has occurred

#### Client version, Server revision & Game ID
 
Client Version: 514 (Multiple different players affected, therefore multiple client version as well)

Server Revision: [1f92103189fd54bc144972ab922a4aab42561e30](https://github.com/Foundation-19/Foundation-19/commit/1f92103189fd54bc144972ab922a4aab42561e30)

Game ID: cjd-aCcP (Multiple other game ID since this can be reproduced in every restart)

#### Issue bingo

- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/vlggms/tegustation-bay12/issues)

**Comment by hry-gh** [comment]
hey, i can't currently replicate this on the provided revision. what was your role/anatomy skill when attempting to remove the organs?

**Comment by PoopyNooges** [comment]
> hey, i can't currently replicate this on the provided revision. what was your role/anatomy skill when attempting to remove the organs?

Okay, I think there's a new update on this issue, apparently, if you clamp bleeders earlier/cut multiple organs at once before removing them, it occurs. If bleeder isn't applied yet, or organ is removed one by one after each separation, it works like normal. I'm not sure yet, this is a new thing, I'll continue to update.


Also my skills for both were masters

**Comment by hry-gh** [comment]
> > hey, i can't currently replicate this on the provided revision. what was your role/anatomy skill when attempting to remove the organs?
> 
> Okay, I think there's a new update on this issue, apparently, if you clamp bleeders earlier/cut multiple organs at once before removing them, it occurs. If bleeder isn't applied yet, or organ is removed one by one after each separation, it works like normal. I'm not sure yet, this is a new thing, I'll continue to update.
> 
> Also my skills for both were masters

yep, can replicate having multiple organs separated totally bones it. i'll look into it, thank you!
